collections.md

```diff
@@ -1377,6 +1377,8 @@ You may optionally pass an integer to `random` to specify how many items you wou
 
     // [2, 4, 5] - (retrieved randomly)
 
+However if Collection has less items than you requested method will fail with `InvalidArgumentException`.
```

Comment:
Just a couple grammar changes.

> However, if **the** Collection has **fewer** items than you requested, **the** method will fail with `InvalidArgumentException.

https://www.grammarly.com/blog/fewer-vs-less/

Reply:
As you say :) I am not native english speaker.

Reply:
no problem, glad to help!